session-5-frontend-react/demo/src/App.js

```diff
@@ -6,64 +6,73 @@ class Tweet extends React.Component {
     this.state = {
       numLike: 0
     };
+    this.buttonOnClick = () => { this.incrementLike(); };
   }
 
   incrementLike() {
-    let previousLike = this.state.numLike;
-    let newState = {
-        numLike: previousLike + 1
-    }
+    const previousLike = this.state.numLike;
+    const newState = {
+      numLike: previousLike + 1
+    };
     this.setState(newState);
   }
 
   render() {
-    let numLike = this.state.numLike;
+    const numLike = this.state.numLike;
     return (
-    <div>
-      <h2> {this.props.tweet} </h2>
-      <button onClick={() => this.incrementLike()}>❤️ {numLike} </button>
-    </div>
+      <div>
+        <h2>{this.props.tweet}</h2>
+        <button onClick={this.buttonOnClick}>Like {numLike}</button>
```

Comment:
Actually, instead of giving up on emoji. It would rather wrap it in `span` and give it a label to introduce the concept of accessibility in writing UI.

Reply:
Up to you. However, folks on Linux (and maybe Windows and even macOS?) may have trouble typing that emoji.

Reply:
I think I am going to introduce that. I will teach how to type emoji in Mac too or they can just copy and paste from README.

Reply:
Sure. Be sure to let me know when you add the section about ARIA to the README, as I would like to review that.